src/main/kotlin/edu/agh/susgame/dto/server/PacketDTO.kt

```diff
@@ -0,0 +1,9 @@
+package edu.agh.susgame.dto.server
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class PacketDTO(
+    val id: Int,
+    val currentMoney: Int,
```

Comment:
To na pewno nie powinno było byc tutaj. currentMoney  do PlayerDTO. Ogólnie packetDTO na ten moment jest niepotrzebne. Robiłem DTO dla każdej klasy i z rozpędu też dla tej walnąłem.

Reply:
Przesunąłem `currentMoney` do `PlayerDTO`

Ale tego `PacketDTO` nie usuwać, co nie? Jest używane w `val packets: List<PacketDTO>` w `ServerSocketMessage#GameState`

Reply:
usunąc całkowicie na frocnie i tak wam się to nie przyda